examples/web/jobs.html

```diff
@@ -1,44 +1,97 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+  <meta charset="UTF-8" />
   <title>Spirit Web Demo — Jobs</title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-  <link href="//fonts.googleapis.com/css?family=Open+Sans:400,600,700&amp;subset=latin,latin-ext" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&amp;subset=latin,latin-ext" rel="stylesheet" />
```

Comment:
We might switch to css2 api here as with Inter, but it could be done in the future.